### PR TITLE
added unit tests to helper and workflow classes in ingestion

### PR DESCRIPTION
**Pull request by parthp2107**
### Describe your changes :

I worked on adding unit tests to helper and workflow classes in ingestion. Fixes issue-#338

#
### Type of change :

- [x] Bug fix
- [x] New feature

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [x] I have performed a self-review of my own. 
- [x] I have tagged my reviewers below.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

### CODE REVIEW

File: ingestion/tests/unit/helpers_test.py [hunk 1]
```diff
@@ -68,9 +70,46 @@ def test_create_get_list_databases(self):
...
+        self.assertRaises(APIError, self._client.get_database_service_by_id, mysql_service.id.__root__)
+
+    def test_7_create_messaging_service(self):
+        create_messaging_service = CreateMessagingServiceEntityRequest(
+            name='sample_kafka_test',
+            serviceType='Kafka',
+            brokers=['localhost:9092'],
+            schemaRegistry='http://localhost:8081'
+        )
+        print(create_messaging_service)
```
Comment (ayush-shah) [inline ingestion/tests/unit/helpers_test.py:92]: Can you remove the print statement over here and on other test methods as well?

File: ingestion/tests/unit/workflow_test.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        # if os.path.isfile(file_path):
+        #     with open(file_path) as ingestionFile:
+        #         ingestionData = str(ingestionFile.read())
+        #     if ingestionData is not None:
+        #         print(ingestionData)
+        #         self.assertEqual(ingestionData is not None, True)
+        #     else:
+        #         self.assertEqual(ingestionData is not None, False)
+        # else:
+        #     self.assertEqual(os.path.isfile(file_path), True, 'File Not Present')
```
Comment (ayush-shah) [inline ingestion/tests/unit/workflow_test.py:69]: Can you remove the comments?

File: ingestion/tests/unit/workflow_test.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+        if workflow_config.get('cron'):
+            del workflow_config['cron']
+        workflow = Workflow.create(workflow_config)
+        workflow.execute()
+        workflow.stop()
+        file_path = '/tmp/mysql_test'
+        with open(file_path) as ingestionFile:
+            ingestionData = ingestionFile.read()
+        print(ingestionData)
```
Comment (ayush-shah) [inline ingestion/tests/unit/workflow_test.py:58]: Can you handle this as well?

File: ingestion/tests/unit/helpers_test.py [hunk 4]
```diff
@@ -12,53 +12,54 @@
...
-    def test_get_service_by_id(self):
-        mysql_service = self._client.get_database_service('local_mysql')
-        self.assertEqual(mysql_service.name, 'local_mysql')
+    def test_2_get_service(self):
+        mysql_service = self._client.get_database_service('local_mysql_test')
+        self.assertEqual(mysql_service.name, 'local_mysql_test')
+
+    def test_3_get_service_by_id(self):
+        mysql_service = self._client.get_database_service('local_mysql_test')
+        self.assertEqual(mysql_service.name, 'local_mysql_test')
```
Comment (ayush-shah) [inline ingestion/tests/unit/helpers_test.py:56]: Can you keep assertEqual one per testcase?

File: ingestion/tests/unit/helpers_test.py (hunk 1 above)
Comment (parthp2107) [reply ingestion/tests/unit/helpers_test.py]: Addressed in the latest commit

File: ingestion/tests/unit/workflow_test.py (hunk 2 above)
Comment (parthp2107) [reply ingestion/tests/unit/workflow_test.py]: Addressed in the latest commit

File: ingestion/tests/unit/workflow_test.py (hunk 3 above)
Comment (parthp2107) [reply ingestion/tests/unit/workflow_test.py]: Addressed in the latest commit

File: ingestion/tests/unit/helpers_test.py (hunk 4 above)
Comment (parthp2107) [reply ingestion/tests/unit/helpers_test.py:56]: Addressed in the latest commit

Comment (ayush-shah) [reply ingestion/tests/unit/helpers_test.py]: @parthp2107 Few tests aren't passing in helpers_test.py
can you take a look at it?

Comment (parthp2107) [reply ingestion/tests/unit/helpers_test.py]: It is now fixed. Addressed in the latest commit

**Review by ayush-shah — APPROVED** [review]

**Pull request merged**